Removes register view, use single LoginView

michaeljcollinsuk opened the pull request:
- Update LoginView to get or create a User and send magic link email
- Remove RegisterView and RegisterForm

symroe reviewed (approved):
Could you also change the URL in the main nav that I made in a commit to main a little while ago?

michaeljcollinsuk commented:
> Could you also change the URL in the main nav that I made in a commit to main a little while ago?

Ah was just adding this then saw your comment. Rebased to pull in your commit and updated, with a logout link too
